Loop/Models/LoopConstants.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LoopConstants.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 10/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+
+struct LoopConstants {
+    
+    // Input field bounds
+    
+    static let maxCarbEntryQuantity = HKQuantity(unit: .gram(), doubleValue: 250)
+    
+    static let validManualGlucoseEntryRange = HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 10)...HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 600)
+
+    
+    // MARK - Display settings
+
+    static let minimumChartWidthPerHour: CGFloat = 50
```

Comment:
The LoopUI framework is not where all the UI code for Loop goes; i.e. most of the UI code in the Loop app is in the Loop target. LoopUI is for code that for UI code that is *shared* between the Loop executables, such as the extension, and Loop itself.  So it might not go there specifically because it is UI specific.

However, these constants *are* shared, so I could see them being split and moved into LoopCore and LoopUI for that reason. I'll take a look at that.

Reply:
Split up into LoopConstants (shared), LoopConstants (Loop only), and ChartConstants (LoopUI shared)